application/consumer-hazelcast-quarkus/src/main/java/webapp/tier/service/HazelcastConnectionWrapper.java

```diff
@@ -9,7 +9,7 @@
 import com.hazelcast.core.HazelcastInstance;
 
 @ApplicationScoped
-public class HazelcastService {
+public class HazelcastConnectionWrapper {
```

Comment:
Codacy found an issue: [The utility class name 'HazelcastConnectionWrapper' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5937188)